src/types/dataTypes/drupal/node.ts

```diff
@@ -267,6 +270,136 @@ export interface NodePersonProfile extends DrupalNode {
   field_suffix: string
 }
 
+/** A individual event published by a facility.
+ *
+ *  @see https://prod.cms.va.gov/admin/structure/types/manage/event/fields
+ */
+export interface NodeEvent extends DrupalNode {
+  field_additional_information_abo: {
+    value: string
+    format: string
+    processed: string
+  }
+  field_address?: {
+    additionalName?: string | null
+    addressLine1?: string
+    addressLine2?: string
+    administrativeArea?: string
+    countryCode?: string
+    dependentLocality?: string | null
+    familyName?: string | null
+    givenName?: string | null
+    langcode?: string
+    locality?: string
+    organization?: string | null
+    postalCode?: string | null
+    sortingCode?: string | null
+  }
+  field_location_humanreadable: string
+  field_event_cta: string
+  field_event_cost: string
+  field_datetime_range_timezone: DateTimeRange[]
+
+  field_facility_location: FacilityLocation
+  field_featured: boolean
+  field_body: {
+    value: string
+    format: string
+    processed: string
+  }
+  field_include_registration_info: boolean
+  field_location_type: string
+  field_order: string
+  field_publish_to_outreach_cal: boolean
+  field_event_registrationrequired: boolean
+
+  field_description: string
+  field_link: fieldLink
+  field_url_of_an_online_event: urlOfOnlineEvent
+  field_listing: fieldEventListing
+}
+
+interface fieldLink {
+  url: {
+    path: string
+  }
+}
+
+export interface DateTimeRange {
+  value: string
+  end_value: string
+  duration: number
+  rrule: number
+  rrule_index: number
+  timezone: string
+}
+
+export interface urlOfOnlineEvent {
+  uri: string
+  title: string
+  options: unknown
+}
+
+export interface fieldEventListing {
+  type: string
+  id: string
+  drupal_internal__nid: number
+  langcode: string
+  status: boolean
+  title: string
+  created: string
+  changed: string
+  breadcrumbs: BreadcrumbItem[]
+  moderation_state: string
+  path: {
+    alias: string
+    pid: number
+    langcode: string
+  }
+  field_description: string
+  field_enforce_unique_combo: boolean
+  field_intro_text: string
+  links: {
+    self: {
+      href: string
+    }
+  }
+  resourceIdObjMeta: {
+    drupal_internal__target_id: number
+  }
+  node_type: {
+    type: string
+    id: string
+    resourceIdObjMeta: unknown
+  }
+}
+
+export interface FacilityLocation extends DrupalNode {
```

Comment:
similarly, I think there may be some duplicated types here. We probably want to be using `NodeHealthCareLocalFacility` (line 130ish). it appears to come through the included json response on events as: `node--health_care_local_facility`